src/audit.ts

```diff
@@ -101,6 +137,18 @@ const collectYarnAuditResults: AuditResultsCollector = async stdout => {
   });
 };
 
+const toMapOfFindings = (findings: Finding[]): Record<string, Finding> => {
```

Comment:
🐘 in `Record<string` and `1Omit<Npm7Vulnerability`, does the `<` mean anything special, or is is just a character in the variable name? I'm just curious